conda_forge_tick/utils.py

```diff
@@ -679,12 +679,12 @@ def _fetch_static_repo(name, dest):
 
 def populate_feedstock_attributes(
     name: str,
-    sub_graph: typing.MutableMapping,
+    sub_graph: AttrsTypedDict_,
```

Comment:
```suggestion
    sub_graph: "AttrsTypedDict_",
```

Either quote this, or import AttrsTypedDict outside of a `typing.TYPE_CHECKING` blockj